sklearn/cluster/_agglomerative.py

```diff
@@ -743,6 +743,11 @@ class AgglomerativeClustering(ClusterMixin, BaseEstimator):
 
         .. versionadded:: 0.21
 
+    return_distance : bool, default=False
+        Computes distances even if no distance_threshold is used. This can be
```

Comment:
May need to rewrap to keep line length < 80:

```suggestion
        Computes distances between clusters even if no `distance_threshold` is set. This can be
```